tests/datadriven/test_reads.py

```diff
@@ -24,7 +64,133 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAReadGroupSet
 
-    def testFixMe(self):
+    def testGetReadAlignmentsBothRefs(self):
+        # test that querying by both referenceName and referenceId fails
+        with self.assertRaises(exceptions.BadReadsSearchRequestBothRefs):
+            readGroupSet = self._gaObject
+            for readGroup in readGroupSet.getReadGroups():
+                list(readGroup.getReadAlignments("a", 5, 0))
+
+    def testGetReadAlignments(self):
+        # test that searching with no arguments succeeds
+        readGroupSet = self._gaObject
+        for readGroup in readGroupSet.getReadGroups():
+            alignments = list(readGroup.getReadAlignments())
+            readGroupInfo = self._readGroupInfos[readGroup.getSamFilePath()]
+            for i, gaAlignment in enumerate(alignments):
```

Comment:
We could also use
```python
gaAlignment, pysamAlignment in zip(alignments, readGroupInfo.reads):
```
this is usually regarded as a bit more Pythonic that using an index variable. I don't feel strongly about it though, this code is still very clear.